membership_file/tests/tests_model.py

```diff
@@ -24,10 +25,44 @@ def test_memberlogfield_display(self):
         memberlogfield = MemberLogField(id=2, field="name", old_value="Bob", new_value="Charlie")
         self.assertEqual(str(memberlogfield), f"name was updated: <Bob> -> <Charlie> (2)")
 
+    def test_is_considered_member_active_years(self):
+        """ Tests member.is_considered_member when active years are present """
+        self.assertTrue(Member.objects.get(id=1).is_considered_member())
+        self.assertFalse(Member.objects.get(id=2).is_considered_member())
+        # Member is deregistered so should not be member, even though there is a active membesrhip connected
+        # (why this scenario would occur in real life I don't know, but let's set consistent behavior)
```

Comment:
I suppose it could be possible that someone explicitly deregisters halfway through the year.